doc/changes/changelog.md

```diff
@@ -0,0 +1,4 @@
+# Changes
+
+* [0.2.0](changes_0.2.0.md)
```

Comment:
Please do not forget to add new changes file for the current changes (#9).

Reply:
done